mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

```diff
@@ -118,8 +118,16 @@ The new annotations model is similar to the new labels model and will be applied
 
 There are now two annotation-related fields with the new model, the `annotations` and the output-only `effective_annotations` fields.
 
+### Provider default values shown at plan-time
+
+`project`, `region`, and `zone` fields will now display their values during plan-time instead of the placeholder `(known after apply)` value normally displayed for fields without fixed default values. These values will be taken from either the Terraform resource config file, provider config, or local environment variables, depending on which variables are supplied by the user, matching the existing per-resource functionality for what default values are used in execution of a Terraform plan.
```

Comment:
```suggestion
`project`, `region`, and `zone` fields will now display their values during plan-time instead of the placeholder `(known after apply)` value normally displayed for fields without provider default values. These values will be taken from either the Terraform resource config file, provider config, or local environment variables, depending on which variables are supplied by the user, matching the existing per-resource functionality for what default values are used in execution of a Terraform plan.
```
I understand fixed != API value here, but might as well be specific?

Reply:
I think the problem is it conflicts with the other usage of provider -- since it *does* use the "provder default" value, as in what is configured in the user's provider. I thought about saying "universal default" or similar to mean "regardless of local configuration", but wasn't sure if that became more or less clear than "fixed" on its own

Reply:
That makes sense. How about "fixed Terraform default values" ? If not, I'm fine with just "fixed". Users that care will probably already know from the `(known after apply)` part.

Reply:
That sounds good to me!